controllers/users.js

```diff
@@ -25,7 +25,7 @@ class Users {
     try {
       const user = await User.create({ email: email.trim(), username: username.trim(), hash });
       if (!user) {
-        return res.status(500).send({
+        return res.status(500).json({
```

Comment:
` (!user) `could also be true when the user is not found, so sending a 500 error is not appropriate or true always.